scripts/validate-sitemap-urls.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+request_and_format_sitemap() {
+    curl https://www.pulumi.com/sitemap.xml > sitemap.xml && xmllint --format sitemap.xml > temp_sitemap.xml && mv temp_sitemap.xml sitemap.xml
+}
+
+parse_urls() {
+    cat sitemap.xml | grep -e loc | sed 's|<loc>\(.*\)<\/loc>$|\1|g' | sed 's/www.pulumi.com/www-reg-staging.pulumi-dev.io/g' > site_urls.txt
+}
+
+request_urls() {
+    cat site_urls.txt | xargs -I {} curl -s -o /dev/null -w "%{http_code} %{url_effective}\n" {} | grep -v '200\|301'
```

Comment:
We might want to make the regex a little bit tighter just to make sure we don't accidentally pick up something with 200 or 301 in the URL -- not that we have any such thing yet, but you know, just in case.

Reply:
This might do the trick (of telling `curl` to follow redirects):

```suggestion
    cat site_urls.txt | xargs -I {} curl -L -s -o /dev/null -w "%{http_code} %{url_effective}\n" {} | grep -v '200\|301'
```

It's surfacing a handful of 403s for me now, where it wasn't before.